mmcv/tensorrt/tensorrt_utils.py

```diff
@@ -1,7 +1,91 @@
+import numpy as np
+import onnx
 import tensorrt as trt
 import torch
 
 
+def preprocess_onnx(onnx_model):
+    """Modify onnx model to match with TensorRT plugins in mmcv.
+
+    Args:
+        onnx_model (onnx.ModelProto): Input onnx model.
+
+    Returns:
+        onnx.ModelProto: Modified onnx model.
+    """
+    graph = onnx_model.graph
+    nodes = graph.node
+    initializers = graph.initializer
+    node_dict = {}
+    for node in nodes:
+        node_outputs = node.output
+        for output in node_outputs:
+            if len(output) > 0:
+                node_dict[output] = node
+
+    init_dict = {_.name: _ for _ in initializers}
+
+    def parse_data(name, typ):
+        if name in node_dict:
+            const_node = node_dict[name]
+            assert const_node.op_type == 'Constant'
+            raw_data = const_node.attribute[0].t.raw_data
+        elif name in init_dict:
+            raw_data = init_dict[name].raw_data
+        else:
+            raise ValueError(f'{name} not found in node or initilizer.')
+        return np.frombuffer(raw_data, typ).item()
+
+    nrof_node = len(nodes)
+    for idx in range(nrof_node):
+        node = nodes[idx]
+        node_attributes = node.attribute
+        node_inputs = node.input
+        node_outputs = node.output
+        node_name = node.name
+        # process NonMaxSuppression node
+        if node.op_type == 'NonMaxSuppression':
```

Comment:
Any better design than hardcode?

Reply:
We can use some `Registry` and `build` mechanism to map the onnx node name and process code, just like mmdetection.
But I am not sure if it worth to do. Most official onnx node does not need to do any process. NonMaxSuppression is a rare case.

Reply:
I think we can just leave it this way. Currently,  we only have one special node to process. Besides, `preprocess_onnx` could be removed when the interface between ONNXRuntime and TensorRT is aligned.